psl-core/src/main/java/org/linqs/psl/server/actions/AddTerm.java

```diff
@@ -0,0 +1,25 @@
+package org.linqs.psl.server.actions;
+
+import org.linqs.psl.reasoner.sgd.term.SGDObjectiveTerm;
+import org.linqs.psl.reasoner.sgd.term.SGDStreamingTermStore;
+
+public class AddTerm extends OnlineServerAction{
```

Comment:
I think the existence of this class may just be because of the lag time between our discussion and implementation.
But just in case, this class is at the wrong level of abstraction.
Adding a reasoner term is too level for a "server action".